diff --git b/radiation.cabal a/radiation.cabal
index 14b47af..0eb34ac 100644
--- b/radiation.cabal
+++ a/radiation.cabal
@@ -18,7 +18,7 @@ cabal-version:       >=1.10
 executable radiation
   main-is:             Main.hs
   hs-source-dirs:      src
-  ghc-options:         -O2 -threaded -fprof-auto
+  ghc-options:         -O2
   -- other-modules:       
   other-extensions:    TupleSections, OverloadedStrings
   build-depends:       base, containers, transformers, process, bytestring, attoparsec,
